src/lj_ccall.c

```diff
@@ -986,6 +1020,10 @@ static int ccall_set_args(lua_State *L, CTState *cts, CType *ct,
     MSize n, isfp = 0, isva = 0;
     void *dp, *rp = NULL;
 
+#if LJ_TARGET_S390X
+    MSize onstack = 0;
```

Comment:
Seems like you are using this variable as a boolean. The `MSize` type is confusing.

Reply:
> @abhay1722 Thanks for your quick update. I wonder if it's possible to run the test suite via qemu for this arch. Then we can more easily get this covered by regression testing in the future.

@agentzh Yes it is possible to run the test suite via qemu on s390x. I am checking the same.
Will keep you posted.

Reply:
Yes indeed, I am using the `onstack` variable as a boolean. Replaced `MSize` type with `uint32_t`.
Thanks for the input.